conf/log4j.properties

```diff
@@ -15,15 +15,9 @@
 # limitations under the License.
 #
 
-log4j.rootCategory=INFO, RollingAppender
+log4j.rootCategory=INFO
 log4j.appender.console=org.apache.log4j.ConsoleAppender
 log4j.appender.console.Threshold=INFO
 log4j.appender.console.target=System.err
 log4j.appender.console.layout=org.apache.log4j.PatternLayout
 log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
-log4j.appender.RollingAppender=org.apache.log4j.RollingFileAppender
```

Comment:
Why do we remove them?

Reply:
Thank you very much for your help to review the code! I will revert this part of the change. When I was testing locally, I modified this file.